transcendence-app/src/game/dto/create-game.dto.ts

```diff
@@ -0,0 +1,28 @@
+import { IsNotEmpty, IsNumber, IsString } from 'class-validator';
+import { GameMode } from '../../../src/shared/enums/game-mode.enum';
+
+export class CreateGameDto {
+  @IsString()
+  @IsNotEmpty()
+  playerOneUsername!: string;
+
+  @IsString()
+  @IsNotEmpty()
+  playerTwoUsername!: string;
+
+  @IsNumber()
```

Comment:
In what? Seconds? Minutes? Hindu eons?

Reply:
I think second would be ok. I will change the name to `gameDurationInSeconds`